Dockerfile

```diff
@@ -1,12 +1,26 @@
-# Use distroless as minimal base image to package the manager binary
-# Refer to https://github.com/GoogleContainerTools/distroless for more details
+
+# TODO: this probably needs to be a separate Dockerfile specifically for konflux (like we currently have for cpaas)
```

Comment:
definitely! if we change this to be built FROM an internal container image, it won't work on prow anymore. or we parametrize the build container image